zh_tw/news/_posts/2022-11-24-ruby-2-7-7-released.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "Ruby 2.7.7 發布"
+author: "usa"
+translator: "Bear Su"
+date: 2022-11-24 12:00:00 +0000
+lang: zh_tw
+---
+
+Ruby 2.7.7 已經發布了。
+
+本次發布版本包含安全性修正。
+細節請參考下列內容。
+
+* [CVE-2021-33621: CGI 存在 HTTP 響應切分風險]({%link zh_tw/news/_posts/2022-11-22-http-response-splitting-in-cgi-cve-2021-33621.md %})
```

Comment:
Looks the file doesn't exist, please use the en version https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2022-11-22-http-response-splitting-in-cgi-cve-2021-33621.md as of now.

Reply:
After the PR #3109 has been merged, the file exists. Should I still change to use the en version?

Reply:
👍